Yosemite/Yosemite/PointOfSale/Items/PointOfSaleBarcodeScanService.swift

```diff
@@ -18,13 +18,15 @@ public final class PointOfSaleBarcodeScanService: PointOfSaleBarcodeScanServiceP
     private let productsRemote: ProductsRemoteProtocol
     private let currencyFormatter: CurrencyFormatter
     private let siteID: Int64
+    private let itemMapper: PointOfSaleItemMapper
 
     init (siteID: Int64,
           productsRemote: ProductsRemoteProtocol,
           currencySettings: CurrencySettings) {
         self.siteID = siteID
         self.productsRemote = productsRemote
         self.currencyFormatter = CurrencyFormatter(currencySettings: currencySettings)
+        self.itemMapper = PointOfSaleItemMapper(currencyFormatter: currencyFormatter)
```

Comment:
nit: now that `currencyFormatter` is moved to the mapper, the property could be removed from the service.
```suggestion
        self.itemMapper = PointOfSaleItemMapper(currencyFormatter: currencyFormatter)
```

Reply:
We can probably just pass the settings through as well. Thanks, I'll sort this.

Reply:
8617fbbc